documentation/smg/12_deploying_releases_to_GitHub.md

```diff
@@ -0,0 +1,89 @@
+# Deploying releases to GitHub
+
+## ci-release-deploy Repository
+
+A repository
+[ci-release-deploy](https://github.com/pace-neutrons/ci-release-deploy)
+has been created to hold the code that implements the deployment pipeline.
+This decision was made so that code is not duplicated between Herbert and Horace
```

Comment:
This should be rephrased to discuss the decision not that the decision was made. The decision is an ADR, the consequences for someone using the system are SMG.

So the focus flips to be "the extra GitHub repo enables fast checkout times and removes the need for duplicate release code in both the Horace and Herbert repositories."